app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, src, dest = command.split()
+
+    if cmd == "mv" and dest.endswith("/"):
+        os.makedirs(dest, exist_ok=True)
+        dest = os.path.join(dest, os.path.basename(src))
+    else:
+        dest_dir = os.path.dirname(dest)
+        if dest_dir:
+            os.makedirs(dest_dir, exist_ok=True)
```

Comment:
I'm a bit unclear about the purpose of this `else`. If the command isn't `mv`, the function should terminate